ukbb_qc/release/vcf_data_dicts.py

```diff
@@ -0,0 +1,625 @@
+            },
+            "age_hist_hom_n_smaller": {
+                "Description": "Count of age values falling below lowest histogram bin edge for homozygous alternate individuals.",
+            },
+            "age_hist_hom_n_larger": {
+                "Description": "Count of age values falling above highest histogram bin edge for homozygous alternate individuals.",
+            },
+            "popmax": {"Description": "Population with maximum allele frequency."},
+            "AC_popmax": {
+                "Description": "Allele count in the population with the maximum allele frequency.",
+            },
+            "AN_popmax": {
+                "Description": "Total number of alleles in the population with the maximum allele frequency.",
+            },
+            "AF_popmax": {
+                "Description": "Maximum allele frequency across populations.",
+            },
+            "nhomalt_popmax": {
+                "Description": "Count of homozygous individuals in the population with the maximum allele frequency.",
+            },
+            "gq_hist_alt_bin_freq": {
+                "Description": "Histogram for GQ in heterozygous individuals calculated on high quality genotypes; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100.",
+            },
+            "gq_hist_all_bin_freq": {
+                "Description": "Histogram for GQ calculated on high quality genotypes; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100.",
+            },
+            "dp_hist_alt_bin_freq": {
+                "Description": "Histogram for DP in heterozygous individuals calculated on high quality genotypes; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100.",
+            },
+            "dp_hist_alt_n_larger": {
+                "Description": "Count of DP values falling above highest histogram bin edge DP in heterozygous individuals calculated on high quality genotypes.",
+            },
+            "dp_hist_all_bin_freq": {
+                "Description": "Histogram for DP calculated on high quality genotypes; bin edges are: 0|5|10|15|20|25|30|35|40|45|50|55|60|65|70|75|80|85|90|95|100.",
+            },
+            "dp_hist_all_n_larger": {
+                "Description": "Count of DP values falling above highest histogram bin edge DP calculated on high quality genotypes.",
+            },
+            "ab_hist_alt_bin_freq": {
+                "Description": "Histogram for AB in heterozygous individuals calculated on high quality genotypes; bin edges are: 0.00|0.05|0.10|0.15|0.20|0.25|0.30|0.35|0.40|0.45|0.50|0.55|0.60|0.65|0.70|0.75|0.80|0.85|0.90|0.95|1.00.",
+            },
+            "gq_hist_alt_raw_bin_freq": {
```

Comment:
These raw histograms need to have some additional text saying "before removing low-confidence genotypes", as the descriptions are exactly the same as the descriptions for the non-raw histograms.